codalab/lib/codalab_manager.py

```diff
@@ -212,17 +212,17 @@ def init_config(self, dry_run=False):
 
         return config
 
-    @property
+    @property  # type: ignore
```

Comment:
Why is `ignore` needed here?

Reply:
this is the suggested way to resolve warnings of Decorator type not supported